modules/custom-tuning-specification.adoc

```diff
@@ -174,3 +174,47 @@ To minimize node reboots, label the target nodes with a label the machine config
 //     matchLabels:
 //       node-role.kubernetes.io/worker-custom: ""
 // EOF
+
+*Cloud provider specific TuneD profiles*
+
+With this functionality, all Cloud provider specific nodes can conveniently
+be assigned TuneD profile specifically tailored to a given Cloud provider
+on {product-title} cluster without the need for adding any additional node
+labels or grouping nodes into Machine Config Pools.
+
+This functionality takes advantage of `spec.providerID` node object values
+in the form of `<cloud-provider>://<cloud-provider-specific-id>` and writes
+file `/var/lib/tuned/provider` with the value `<cloud-provider>` in NTO
+operand containers.  The content of this file is then used by TuneD to load
+`provider-<cloud-provider>` profile if such profile exists.
+
+The `openshift` profile that both `openshift-control-plane` and
+`openshift-node` profiles inherit settings from has been adjusted to take
```

Comment:
profiles inherit settings from is now updated to use this functionality through the use of conditional profile loading.